source/code-snippets/usage-examples/updateMany.js

```diff
@@ -0,0 +1,35 @@
+// ignored first line
+const { MongoClient } = require("mongodb");
+
+// Replace the following with your MongoDB deployment's connection string.
+const uri =
+  "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority";
+
+const client = new MongoClient(uri);
+
+async function run() {
+  try {
+    await client.connect();
+
+    const database = client.db("sample_mflix");
+    const collection = database.collection("movies");
+
+    // create a filter for movies to update
+    const filter = { rated: "G" };
+    const options = {};
+    options.upsert = true; // create a document if no documents match the filter
```

Comment:
I feel like this example is very similar to the updateOne example. Maybe we could leave out the upsert here since in this use case it doesn't really do anything anyway? Including it could be confusing to readers.

Reply:
*  This example is actually similar to 'update one' on purpose
* I'd like to keep some additional options to adhere to the M.O.R (and sometimes .E) paradigm. 
* In my opinion, upsert is the best one for `updateMany` (see: https://mongodb.github.io/node-mongodb-native/3.4/api/Collection.html#updateMany) because we don't want to talk about write concern at the method level and we don't want to bypass-document validation. 
* upsert here does create documents if there are no 'g' rated movies